I tried to reproduce, but actually my job fails at step1, I get an error directly in the DAS query:

Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_9_2_X_2017-06-14-2300/bin/slc6_amd64_gcc530/das-selected-lumis.py", line 44, in <module>
    process_lumi(lumi_data)
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_9_2_X_2017-06-14-2300/bin/slc6_amd64_gcc530/das-selected-lumis.py", line 32, in process_lumi
    lumi_nums = lumi_info['lumi'][0]['number']
KeyError: 0

It looks fishy again, could you please restart the test a last time

--- a/DQM/SiPixelPhase1TrackClusters/interface/SiPixelPhase1TrackClusters.h
+++ b/DQM/SiPixelPhase1TrackClusters/interface/SiPixelPhase1TrackClusters.h
@@ -29,7 +29,7 @@ class SiPixelPhase1TrackClusters : public SiPixelPhase1Base {
 
     NTRACKS,
     NTRACKS_VOLUME,
-    SIZE_VS_ETA,
+    ONTRACK_SIZE_VS_ETA
 
   };
 

--- a/DQM/SiPixelPhase1TrackClusters/python/SiPixelPhase1TrackClusters_cfi.py
+++ b/DQM/SiPixelPhase1TrackClusters/python/SiPixelPhase1TrackClusters_cfi.py
@@ -164,8 +164,8 @@ SiPixelPhase1TrackClustersNTracksInVolume = DefaultHistoTrack.clone(
 
 )
 
-SiPixelPhase1ClustersSizeVsEta = DefaultHistoTrack.clone(
-  name = "sizeyvseta",
+SiPixelPhase1ClustersSizeVsEtaOnTrack = DefaultHistoTrack.clone(
+  name = "sizeyvseta_on_track",
   title = "Cluster Size along Beamline vs. Cluster position #eta (OnTrack)",
   xlabel = "Cluster #eta",
   ylabel = "length [pixels]",
@@ -195,7 +195,7 @@ SiPixelPhase1TrackClustersConf = cms.VPSet(
 
   SiPixelPhase1TrackClustersNTracks,
   SiPixelPhase1TrackClustersNTracksInVolume,
-  SiPixelPhase1ClustersSizeVsEta
+  SiPixelPhase1ClustersSizeVsEtaOnTrack
 )
 
 
@@ -210,3 +210,5 @@ SiPixelPhase1TrackClustersHarvester = DQMEDHarvester("SiPixelPhase1Harvester",
         histograms = SiPixelPhase1TrackClustersConf,
         geometry = SiPixelPhase1Geometry
 )
+
+

--- a/DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc
+++ b/DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc
@@ -157,7 +157,7 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
         histo[ONTRACK_SIZE      ].fill(double(cluster.size()  ), id, &iEvent);
         histo[ONTRACK_POSITION_B].fill(clustgp.z(),   clustgp.phi(),   id, &iEvent);
         histo[ONTRACK_POSITION_F].fill(clustgp.x(),   clustgp.y(),     id, &iEvent);
-	histo[SIZE_VS_ETA].fill(clustgp.eta(), cluster.sizeY(), id, &iEvent);
+	histo[ONTRACK_SIZE_VS_ETA].fill(clustgp.eta(), cluster.sizeY(), id, &iEvent);
       } else {
         histo[OFFTRACK_NCLUSTERS ].fill(id, &iEvent);
         histo[OFFTRACK_CHARGE    ].fill(double(cluster.charge()), id, &iEvent);

